docs/quickstart.rst

```diff
@@ -11,6 +11,14 @@ Install a package from `PyPI`_:
     [...]
     Successfully installed SomePackage
 
+Install a package downloaded from `PyPI`_ or got elsewhere. This is useful if the target machine does not have a network connection:
+
+::
+
+  $ pip install SomePackage.tar.gz
```

Comment:
I'm fine either way. Should I change it and update the PR? Any other changes? Do you care are commits squashed at the end?